tests/main/regression-home-snap-root-owned/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Regression test that ensures that $HOME/snap is not root owned for sudo commands
+
+systems: [-ubuntu-core-16-64]
```

Comment:
I recall this fails on core but it would be worth adding a comment why. I don't quite understand why it fails there.